Nikita_Konkov_Application/Selenium_Basics/Selenium_Basics/EpamHowWeDoItOurWorkNavigationTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_Basics
+{
+    [TestFixture]
+    public class EpamHowWeDoItOurWorkNavigationTest
+    {
+        private IWebDriver webDriver = new ChromeDriver();
+        private string _epamHowWeDoItLink;
+        private string _epamOurWorkLink;
+
+        [SetUp]
+        public void Setup()
+        {
+            _epamHowWeDoItLink = "https://www.epam.com/how-we-do-it";
```

Comment:
Why do you need text constants in setup?

Reply:
To use it in the test. Not sure that I understood the question right.

Reply:
text constants should be placed int class variables of the test's variables. You placed urls in setup. Constant is not changes, so why we need setup to this variables?